website/docs/icons/library/index.js

```diff
@@ -34,11 +34,21 @@ export default class Index extends Component {
 
   get filteredIcons() {
     if (this.searchQuery) {
-      return this.allIcons.filter(
-        (i) =>
-          i.size === this.selectedIconSize &&
-          i.searchable.indexOf(this.searchQuery) !== -1
-      );
+      // check if the query is for an exact match (prefixed with `icon:`)
+      if (this.searchQuery.match(/^icon:/)) {
+        const iconValue = this.searchQuery.replace(/^icon:/, '');
+        const icon = this.allIcons.find((i) => {
+          // we want to match both `youtube` as well as `youtube-24`
```

Comment:
I'm curious about the 2nd part of this matching a string with the size in it. If `youtube-16` is passed in do we need to show that version of the icon?